src/log_config.py

```diff
@@ -5,7 +5,7 @@
 from verbose_logging_helper import VerboseLoggingHelper
 
 DEFAULT_LOG_FILE = 'logs/app.log'
-FORMATTER = logging.Formatter('%(asctime)s - %(threadName)-9s - %(message)s')
+FORMATTER = logging.Formatter('%(asctime)s - %(threadName)-9s - %(levelname)s - %(message)s')
```

Comment:
IMHO it would be interesting to see what kind of log message is currently displayed.

Reply:
@vkresch 

The app.log is for the user. The verbose log is for the maintainers. I think level information in the app.log is not very relevant.

Reply:
ok, leave the levelname and we see how users react on it.